src/webvr-manager.js

```diff
@@ -430,13 +431,19 @@ WebVRManager.prototype.releaseOrientationLock_ = function() {
 
 WebVRManager.prototype.requestFullscreen_ = function() {
   var canvas = document.body;
+  var vrDisplayOptions = {vrDisplay: this.hmd};
   //var canvas = this.renderer.domElement;
   if (canvas.requestFullscreen) {
     canvas.requestFullscreen();
   } else if (canvas.mozRequestFullScreen) {
-    canvas.mozRequestFullScreen({vrDisplay: this.hmd});
+    // Stable Firefox for Android does not like it when 
```

Comment:
IMO, might as well do this for Chrome too. could just handle this [above](#diff-f94dd7be6e60ebe39087d9f5f4895a82R434).